packages/editor-ui/src/stores/collaboration.store.ts

```diff
@@ -14,37 +14,48 @@ export const useCollaborationStore = defineStore(STORES.COLLABORATION, () => {
 	const workflowStore = useWorkflowsStore();
 
 	const usersForWorkflows = ref<ActiveUsersForWorkflows>({});
+	const removeEventListener = ref<(() => void) | null>(null);
```

Comment:
Nit: I find this name a bit confusing (imperative form makes me think it's a function or that it's doing something)

Reply:
Yeah, it's pretty confusing. Updated the name. What do you think?

Reply:
Better, thanks